libraries/aws/shadow/cbmc/proofs/Shadow_GetTopicString/Shadow_GetTopicString_harness.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file Shadow_GetTopicString_harness.c
+ * @brief Implements the proof harness for Shadow_GetTopicString function.
+ */
+
+#include "shadow.h"
+#include "shadow_cbmc_state.h"
+
+#ifndef TOPIC_STRING_LENGTH_MAX
+    #define TOPIC_STRING_LENGTH_MAX    ( 128U )
+#endif
+
+#define TEST_THING_NAME          "testThingName"
+#define TEST_THING_NAME_LENGTH   ( sizeof( TEST_THING_NAME ) - 1U )
+
+void harness()
+{
+    ShadowStatus_t shadowStatus = SHADOW_SUCCESS;
+    char * topicBuffer;
+    uint16_t bufferSize;
+    uint16_t outLength = 0;
```

Comment:
This should not be set,
```suggestion
    uint16_t outLength;
```

Reply:
please review [this commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1134/commits/87cb11ae1496f618c50d49ad2ed7fd5c3630a3ec) for the update.